internal/flink/command_statement_exception_list.go

```diff
@@ -46,7 +46,7 @@ func (c *command) statementExceptionList(cmd *cobra.Command, args []string) erro
 		list.Add(&exceptionOut{
 			Name:       exception.GetName(),
 			Timestamp:  exception.GetTimestamp(),
-			StackTrace: exception.GetStacktrace(),
+			StackTrace: exception.GetMessage(),
```

Comment:
Is the API still returning a stack trace, or does "message" mean something else?

Reply:
No it's not a stack trace anymore, it's now a user-friendly message explaining the error (to avoid leaking internals)

Reply:
Got it. This is another breaking change so let's make sure it gets documented here: https://confluentinc.atlassian.net/wiki/spaces/DPI/pages/2907775380/Feature+Flag+Breaking+Changes

Reply:
Yeah I'll add it to the doc (once I know the version number) 👍

Reply:
I've updated the table with version 3.51.0 since that should be the next one